src/components/Account/AccountModule.js

```diff
@@ -167,78 +124,49 @@ function AccountModule() {
           display={compactMode ? 'icon' : 'all'}
         />
       )}
-
       <HeaderPopover
-        animateHeight={animate}
+        direction={direction}
         heading={screen.title}
-        height={screen.height}
-        width={51 * GU}
+        keys={({ screenId }) => screenId + activating + activationError.name}
         onClose={handlePopoverClose}
+        onOpen={open}
         opener={buttonRef.current}
+        screenId={screenId}
+        screenData={{
+          account,
+          // This is needed because use-wallet throws an error when the
+          // activation fails before React updates the state of `activating`.
+          // A future version of use-wallet might return an
+          // `activationError` object instead, making this unnecessary.
+          activating: screen.id === 'error' ? null : activatingDelayed,
```

Comment:
Do we still need this now with use-wallet 0.8?

Reply:
Nope, this won't be needed, I'm pretty sure. :) Let's update to 0.8!

Reply:
Done in https://github.com/aragon/court-dashboard/pull/375/commits/8bd72dc57eb044f5b7f3f20e7749fc95ab76749e